lib/roo_on_rails/routemaster/publisher.rb

```diff
@@ -4,15 +4,17 @@
 module RooOnRails
   module Routemaster
     class Publisher
-      attr_reader :model, :event
+      attr_reader :model, :event, :force_publish
 
-      def initialize(model, event, client: ::Routemaster::Client)
+      def initialize(model, event, client: ::Routemaster::Client, force_publish: false)
         @model = model
         @event = event
         @client = client
+        @force_publish = force_publish
       end
 
       def publish?
```

Comment:
Is there a reason you have this as a class instance variable, rather than an argument to `publish?`?

I'd imagine that `publisher.publish?(force: true)` would read better than having to scan back to wherever the publisher is initialized to know whether or not it'll be published or not.

Reply:
I just felt it better fits the current code since the `publish?` method is called through `will_publish?` method so it will have to receive `force_publish` parameter too. But `will_publish?` method is actually not forced to publish (it checks in config whether publishing is enabled). But if you think it makes more sense that way, I can definitely change it.